packages/workflow/src/workflow.ts

```diff
@@ -127,8 +152,9 @@ function timerNextHandler(input: TimerInput) {
  *
  * @param ms sleep duration - number of milliseconds or {@link https://www.npmjs.com/package/ms | ms-formatted string}.
  * If given a negative number or 0, value will be set to 1.
+ * @param summary a short summary/description of the timer. Can serve as a timer ID.
  */
-export function sleep(ms: Duration): Promise<void> {
+export function sleep(ms: Duration, summary?: string): Promise<void> {
```

Comment:
I'm worried this will evolve badly. What if we then need to add `description`, and then other properties? Instead, I think I'd define a `TimerOptions` type, and take this as a second argument here. Then, we can easily add whatever properties we need.

Reply:
Out of curiosity, what do we do in other SDKs?